cargo-{login,publish,package,yank,owner}: add page; cargo-logout: update page (#11337)
tldr-pages/tldr: pages/common/cargo-publish.md

# cargo publish

> Upload a package to a registry.
> Note: you have to add an authentication token using `cargo login` before publishing a package.
> More information: <https://doc.rust-lang.org/cargo/commands/cargo-publish.html>.

- Perform checks, create a `.crate` file and upload it to the registry:

`cargo publish`

> - Perform checks, create a `.crate` file but don't upload it (equivalent of `cargo package`):

Reviewer: ```suggestion
- Perform checks, create a `.crate` file without uploading it (equivalent of `cargo package`):
```

Avoiding the `don't`

Author: Why do you want to avoid using `don't`?
